components/ChallengeMaterial.tsx

```diff
@@ -51,18 +59,26 @@ export const ChallengeTitleCard: React.FC<ChallengeTitleCardProps> = props => {
   )
 }
 
-export const ChallengeQuestionCard: React.FC<ChallengeQuestionCardProps> = props => {
-  const content = (
-    <div className="card-body">
-      <h1 className="challenge-question-card__title">{props.title}</h1>
-      <div>
-        <p className="challenge-question-card__question bg-light p-3 mt-3">
-          {props.question}
-        </p>
+export const ChallengeQuestionCard: React.FC<ChallengeQuestionCardProps> = ({
+  currentChallenge
+}) => {
+  return (
+    <div className="card shadow-sm border-0">
+      <div className="card-body challenge-question">
+        <h1 data-testid="challenge-question-title" className="card-title">
```

Comment:
data-testid?

Reply:
to make this selectable during test: https://github.com/garageScript/c0d3-app/pull/118/files?file-filters%5B%5D=.js&file-filters%5B%5D=.scss&file-filters%5B%5D=.tsx#diff-7e495cc52c6d8a8ca0f0836a2c6a1f5dR42